X/X13as/build_tarballs.jl

```diff
@@ -0,0 +1,80 @@
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+########## First, build and install the ascii version
+pushd x13as_asciisrc
+###  Fix a bug in adpdrg.f in x13as_asciisrc v1.1.b60
+sed -i -e "s|'\,T|'|" adpdrg.f
+if [[ "${target}" == *-apple-* ]]
+then
+    # the -static flag doesn't work in macos; also disable -s linker flag
+    sed -e "s/\-static//" -i makefile.gf
+    make -j${nproc} -f makefile.gf PROGRAM="x13as_ascii${exeext}" LDFLAGS=
+else
+    make -j${nproc} -f makefile.gf PROGRAM="x13as_ascii${exeext}"
+fi
+mkdir -p ${bindir}
+install -Dvm 755 "x13as_ascii${exeext}" "${bindir}/x13as_ascii${exeext}"
+popd
+########## Second, build and install the html version
+pushd x13as_htmlsrc
+###  They left the debug flags here for some reason
+sed -e "s/FFLAGS.*/FFLAGS=-O2/" -i makefile.gf
+if [[ "${target}" == *-apple-* ]]
+then
+    # the -static flag doesn't work in macos; also disable -s linker flag
+    sed -e "s/\-static//" -i makefile.gf
+    make -j${nproc} -f makefile.gf PROGRAM="x13as_html${exeext}" LDFLAGS=
+else
+    make -j${nproc} -f makefile.gf PROGRAM="x13as_html${exeext}"
+fi
+install -Dvm 755 "x13as_html${exeext}" "${bindir}/x13as_html${exeext}"
+popd
+########## Third, install the docs and the test examples
+docsdir="${prefix}/docs"
+mkdir -p "${docsdir}"
+install -Dvm 644 "x13as/testairline.spc" "${docsdir}/testairline.spc"
+install -Dvm 644 "x13as/docs/docx13as.pdf" "${docsdir}/docx13as.pdf"
+install -Dvm 644 "x13as/docs/qrefX13ASunix.pdf" "${docsdir}/qrefX13ASunix.pdf"
+install -Dvm 644 "x13as/docs/docX13ASHTML.pdf" "${docsdir}/docX13ASHTML.pdf"
+install -Dvm 644 "x13as/docs/qrefX13ASHTMLunix.pdf" "${docsdir}/qrefX13ASHTMLunix.pdf"
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = expand_gfortran_versions(platforms)
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("x13as_ascii", :x13as_ascii),
+    ExecutableProduct("x13as_html", :x13as_html),
+    FileProduct("docs/testairline.spc", :testairline_spc),
+    FileProduct("docs/docx13as.pdf", :docx13as_pdf),
+    FileProduct("docs/qrefX13ASunix.pdf", :qrefX13ASunix_pdf),
+    FileProduct("docs/docX13ASHTML.pdf", :docX13ASHTML_pdf),
+    FileProduct("docs/qrefX13ASHTMLunix.pdf", :qrefX13ASHTMLunix_pdf)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"13.2.0")
```

Comment:
Please see https://github.com/JuliaPackaging/Yggdrasil/blob/82657bf5b3e19935dddbd39ec0370616fc627209/CONTRIBUTING.md#compatibility-tips.  The fact that this works for libgfortran v3 and v4 (gcc 6 and 7) is a strong evidence that gfortran 13 isn't needed.

Reply:
Thanks for pointing that out. I have removed the `preferred_gcc_version=` option altogether.